packages/clerk-js/src/ui/utils/phoneUtils.test.ts

```diff
@@ -198,4 +199,32 @@ describe('phoneUtils', () => {
       expect(res).toBe('lc');
     });
   });
+
+  describe('getLongestValidCountryCode(value, options)', () => {
+    it('finds valid country code and returns phone number value without country code', () => {
+      const countries = [
```

Comment:
Do we need to mock this? Can we use the actual country map instead for better coverage?

Reply:
@nikos regarding the first point, the function expects a list of countries as shown above in order to iterate over them, thus the mocking. For the latter point, we use a `find` array method for it will return the first it finds in the list. In the above example `us` comes before `do`.

We have to start with 4 digits and then go down because the opposite would result in returning only single digits. It would return `us` every time and never reach `bs` (see above)

Reply:
For the second issue, the current algo will probably return an unwanted result for all us numbers starting from, lets say, "12" (1 is the country code, 2 is the first phone digit).

Reply:
@nikosdouvlis I can confirm this works as expected `+12064563059`. returns `us`